dist/samples/places-autocomplete-addressform/index.js

```diff
@@ -36,42 +48,56 @@ function initAutocomplete() {
 function fillInAddress() {
   // Get the place details from the autocomplete object.
   const place = autocomplete.getPlace();
-
-  for (const component in componentForm) {
-    document.getElementById(component).value = "";
-    document.getElementById(component).disabled = false;
-  }
+  let address1 = "";
+  let postcode = "";
 
   // Get each component of the address from the place details,
   // and then fill-in the corresponding field on the form.
   for (const component of place.address_components) {
```

Comment:
place.address_components could be undefined if the user enters an address autocomplete does not know can presses enter. is the code ready for that, or does it error?